Sources/PointFree/VideoJs.swift

```diff
@@ -0,0 +1,40 @@
+import Css
+import Foundation
+import Html
+import HtmlCssSupport
+
+var videoJsHead: [ChildOf<Element.Head>] {
+  let videoJsAssets: [ChildOf<Element.Head>] = [
+    link([
+      href("https://cdnjs.cloudflare.com/ajax/libs/video.js/7.2.4/alt/video-js-cdn.min.css"),
+      rel(.stylesheet)
+      ]),
+    .init(script([
+      src("https://cdnjs.cloudflare.com/ajax/libs/video.js/7.2.4/video.min.js"),
+      `defer`(true)
+      ]))
+  ]
+
+  return [
+    style(".vjs-subs-caps-button" % display(.none)),
+    script("window.HELP_IMPROVE_VIDEOJS = false;")
+    ]
+    + (Current.envVars.appEnv == .testing ? [] : videoJsAssets)
```

Comment:
I'm not happy with this but not sure what else to do. if we serve up the video.js assets during testing we get non-deterministic screenshots cause sometimes things load in time.

Reply:
Seems fine to me